core/src/main/scala/com/yahoo/maha/core/FilterOperation.scala

```diff
@@ -1368,6 +1377,21 @@ object Filter extends Logging {
     }
   }
 
+  implicit def filterSetJSONR: JSONR[List[Filter]] = new JSONR[List[Filter]] {
```

Comment:
You shouldn't need this.  json4s can serialize and deserialize lists, so you just need to define a JSONR[Filter] which is already there right?

Reply:
json4s can serde ::, which is the super of List.  List 'works' by extension, but it isn't a stable call.

Reply:
If you check the line no 1386, we are already deserializing the List[Filter], why do we need JSONR for List[Filter]? 
```val fil = OrFilter.applyJSON(fieldExtended[List[Filter]]("filterExpressions"))(json)```

Reply:
this just to fix the errors in intellij in FilterOperation class